### DISCUSSION TITLE
Update pt-BR translation

### DISCUSSION THREAD

**Discussion by gabopupo** [aclap-dev/video-downloadhelper#803] [Language & Translation]
[pt_BR.txt](https://github.com/user-attachments/files/15519462/pt_BR.txt)

**Comment by Jota4wd** [comment]
[pt_BR.txt](https://github.com/user-attachments/files/15521990/pt_BR.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translate for that language 🙏
